src/tools/genVersionHeader.pl

```diff
@@ -108,6 +115,7 @@
         $opt_V = $result;
         $vcs = 'Bazaar';
     }
+    $cv = `bzr version-info -q --custom --template='{date}'`;
```

Comment:
AI on ANJ: Check/test bzr.

Reply:
This gives `2017-03-03 18:18:57 -0600` (the last commit on the Base-3.14 BZR branch).